decidim-elections/lib/decidim/elections/seeds.rb

```diff
@@ -12,7 +12,58 @@ def initialize(participatory_space:)
       end
 
       def call
-        # add seeds here
+        component = create_component!
+
+        2.times do
+          create_election!(component:)
+        end
+      end
+
+      def create_component!
+        params = {
+          name: Decidim::Components::Namer.new(participatory_space.organization.available_locales, :elections).i18n_name,
+          published_at: Time.current,
```

Comment:
Let's avoid to automatically publish at this moment, as the frontent does not exists, and action would lead to a `uninitialized constant Decidim::Elections::ElectionsController` error.

```suggestion
          published_at: nil
```